apps/js_urls/utils.py

```diff
@@ -0,0 +1,18 @@
+# Python Standard Library
```

Comment:
Just need one of these labels, not both

Reply:
This file contents moved to `htk/utils/strings.py`